### ISSUE TITLE
About 1.20.4

### ISSUE THREAD

**Issue by vico93** [Pepperoni-Jabroni/MapAtlases#134]
Oh hi there!

First of all, thank you so much for creating such incredible useful mod! Can't leave my modpack without Map Atlases. That said, i want to ask if there still plans to update this mod to 1.20.4/5.

Best regards and thank you again for this amazing mod!

**Comment by dagreenkat** [comment]
I would like to know this too!

**Comment by vico93** [comment]
@TechPro424 is working in a fork of this one to 1.20.4. It still have a couple of bugs (M keybind crashes the game), but the atlas render when right-clicked:

https://github.com/TechPro424/MapAtlases

**Comment by dagreenkat** [comment]
Is TechPro taking over development? i.e. will that updated version be pushed to the main modrinth page when it's fully ready, or should I keep a lookout on github for that? Love to know a sense of timeline if you know it.

**Comment by TechPro424** [comment]
@dagreenkat not too sure about how I'm planning to release it, my current focus is on actually getting the mod to work properly on 1.20.4 
@Pepperoni-Jabroni any particulars you want to state regarding my fork's release? Or should I just make a PR to this repo?

**Comment by TechPro424** [comment]
Seems to work, just waiting for @vico93 to get back to me after testing, as I haven't used the original mod myself

**Comment by dagreenkat** [comment]
I've compiled a version from your changes and it seems like it's better, but not all features are working. Most notably, it should pull up a minimap when you have it in the atlas slot or in your hand, rather than you having to open the map every time to see it. And the zoom on the map should be more adjustable while you're viewing the atlas. IDK how clear that is. So it sort of works! Better than crashing for sure, but still hoping for the basic updates.

**Comment by TechPro424** [comment]
For the minimap
I've figured out what the issue is, but not the reason

So any other problems you want me to fix in the meantime, while I work on this one?

**Comment by dagreenkat** [comment]
<img width="421" alt="Screenshot 2024-03-28 at 11 54 01 AM" src="https://github.com/Pepperoni-Jabroni/MapAtlases/assets/16727025/036cce19-051f-4902-929a-0858631737b9">
<img width="413" alt="Screenshot 2024-03-28 at 11 54 08 AM" src="https://github.com/Pepperoni-Jabroni/MapAtlases/assets/16727025/b8aec84a-54c1-44c5-bf96-c6c39aa0cbce">

Here's an example when opening up the atlas of the map being poorly position. Using the Better Nether Maps mod alongside it, the nether map appears completely empty when in the overworld. The overworld map is still visible in the nether, but the nether map has the same inappropriate zoom level problem. It should default to filling the screen, and also be adjustable with zoom and dragging.
<img width="423" alt="Screenshot 2024-03-28 at 12 05 45 PM" src="https://github.com/Pepperoni-Jabroni/MapAtlases/assets/16727025/a7387a1b-5fb5-41a7-8336-b6e79f2dbc2b">


And some examples of the expected behavior (what we had before 1.20.4):
![IMG_3844](https://github.com/Pepperoni-Jabroni/MapAtlases/assets/16727025/cf87f531-90c7-4e59-935a-a19a9cc0cbf4)

![IMG_3646](https://github.com/Pepperoni-Jabroni/MapAtlases/assets/16727025/150c4fca-37cf-46f5-8f4a-76fc398308da)

Thanks so much!

**Comment by TechPro424** [comment]
![image](https://github.com/Pepperoni-Jabroni/MapAtlases/assets/86620085/fcb5d0db-d97e-4f54-8327-2e0a31f6955f)

This is how I have the atlas when opened
Looks like this because I havent loaded the other chunks yet
But otherwise looks good, anything I'm not noticing here?

**Comment by vico93** [comment]
> ![image](https://private-user-images.githubusercontent.com/86620085/318001685-fcb5d0db-d97e-4f54-8327-2e0a31f6955f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3MjU0MTgsIm5iZiI6MTcxMTcyNTExOCwicGF0aCI6Ii84NjYyMDA4NS8zMTgwMDE2ODUtZmNiNWQwZGItZDk3ZS00ZjU0LTgzMjctMmUwYTMxZjY5NTVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI5VDE1MTE1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNhZjg3NzY0MTVhNGZjZjY5MGJkZTIzZGViZDQ0ZWFhMzA5MGZlOTY1YWRiZjFmNjIzMzc2MWQ0MWQzOTUyMTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-ylu5xdV8vBSfaun3YHpp-E5umtcxvrlVWroiRNhCk4)
> 
> This is how I have the atlas when opened Looks like this because I havent loaded the other chunks yet But otherwise looks good, anything I'm not noticing here?

where can i find the latest .jar?

**Comment by dagreenkat** [comment]
@TechPro424  Yes, you're not able to zoom in or out with the atlas. So it will generate correctly, but you can't for example zoom out on that picture once you've generated more atlas. It might be easier for you to see if you make an atlas with large scale (1:16) maps. Basically, it's stuck at one static level of zoom— you can drag around to see different parts of it if the map is that large, but can't zoom in or out to see a part of the map.

Vico, you'd need to compile from the source that techpro has on the github fork.

**Comment by TechPro424** [comment]
How would you zoom it in the working version? Using the scroll wheel, or a slider?

**Comment by TechPro424** [comment]
Also shall we move this discussion to a new issue in my fork, if you two are ok?

**Comment by vico93** [comment]
> Also shall we move this discussion to a new issue in my fork, if you two are ok?

Yep, but i noticed your fork doesnt have issues enabled. Is this because forks cant have those?

**Comment by TechPro424** [comment]
Enabled issues
